input.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
The contents of this file can also be a variable inside the script itself, no need to include json at all.